app.rb

```diff
@@ -1,137 +1,112 @@
-# include files
-require_relative './person'
-require_relative './teacher'
-require_relative './student'
-require_relative './book'
-require_relative './rental'
-require 'date'
-
-# create apps  class
-class Apps
+require './person'
+require './classroom'
+require './book'
+require './rental'
+require './student'
+require './teacher'
+
+class App
   def initialize
-    @rentals = []
-    @people = []
     @books = []
+    @people = []
+    @rentals = []
+    @classroom_one = Classroom.new('A')
   end
 
-  # list all books
-  def list_books
-    if @books.empty?
-      puts 'No book added yet'
-      return
+  def get_option(user_option)
+    case user_option
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rentals
     end
-    @books.map do |book|
-      puts "Title: \"#{book.title}\", Author: #{book.author}"
+  end
+
+  def list_books
+    puts 'The are no books registered! Please add a student or teacher.' if @books.empty?
+    @books.each do |book|
+      puts "Title: #{book.title} Author: #{book.author}"
     end
   end
 
-  #  list all people
   def list_people
-    if @people.empty?
-      puts 'No person record created yet'
-      return
-    end
+    puts 'The are no people registered! Please add a student or teacher.' if @people.empty?
     @people.map do |person|
-      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      puts "[#{person.class}] Id:#{person.id} Name:#{person.name} Age: #{person.age}"
     end
   end
 
-  # create person
   def create_person
-    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
-    option = gets.chomp
-
-    case option
+    puts 'Do you want to create a student(1) or a teacher(2)?[input number]:'
+    user_option = gets.chomp
+    case user_option
     when '1'
-      create_student
+      puts 'Age:'
+      age = gets.chomp.to_i
+      puts 'Do you have parent permission[Y/N]:'
+      parent_permission = gets.chomp.downcase
+      puts 'Name:'
+      name = gets.chomp
```

Comment:
- [ ] Please keep in mind the Best practices and Solid principles, all requests for input and input validation should've been placed in the main.rb file or classes/methods whose responsibilities are to request or validate input, kindly consider refactoring your code so the input requests are separated from the method, kindly consider doing this in all your methods